packages/flutter/lib/src/widgets/view.dart

```diff
@@ -2,48 +2,84 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:ui' show FlutterView;
+import 'dart:collection';
+import 'dart:ui' show FlutterView, SemanticsUpdate;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
 
 import 'framework.dart';
 import 'lookup_boundary.dart';
 import 'media_query.dart';
 
-/// Injects a [FlutterView] into the tree and makes it available to descendants
-/// within the same [LookupBoundary] via [View.of] and [View.maybeOf].
+/// Bootstraps a render tree that is rendered into the provided [FlutterView].
+///
+/// The content rendered into that view is determined by the provided [child].
+/// Descendants within the same [LookupBoundary] can look up the view they are
+/// rendered into via [View.of] and [View.maybeOf].
 ///
 /// The provided [child] is wrapped in a [MediaQuery] constructed from the given
 /// [view].
 ///
-/// In a future version of Flutter, the functionality of this widget will be
-/// extended to actually bootstrap the render tree that is going to be rendered
-/// into the provided [view]. This will enable rendering content into multiple
-/// [FlutterView]s from a single widget tree.
-///
 /// Each [FlutterView] can be associated with at most one [View] widget in the
 /// widget tree. Two or more [View] widgets configured with the same
 /// [FlutterView] must never exist within the same widget tree at the same time.
-/// Internally, this limitation is enforced by a [GlobalObjectKey] that derives
-/// its identity from the [view] provided to this widget.
+/// This limitation is enforced by a [GlobalObjectKey] that derives its identity
+/// from the [view] provided to this widget.
+///
+/// Since the [View] widget bootstraps its own independent render tree, neither
+/// it nor any of its descendants will insert a [RenderObject] into an existing
+/// render tree. Therefore, the [View] widget can only be used in those parts of
+/// the widget tree where it is not required to participate in the construction
+/// of the surrounding render tree. In practical terms, this means it can
+/// typically be used at the root of the widget tree outside of any other [View]
```

Comment:
This "participate in the construction of the surrounding render tree" gets really technical that I can't understand even after reading multiple times. I'm proposing:
```suggestion
/// render tree. Therefore, the [View] widget can not be used in the regular
/// zones of the widget tree that expects the widget to generate
/// [RenderObject]s. Instead, the [View] widget must be used in special zones
/// that expect *no* [RenderObject]s are generated. In practical terms, this
/// typically means that a [View] can be used as:
///
///   * The root of the widget tree.
///   * The argument of [ViewAnchor.view].
///
/// Additionally, a [View] can also be wrapped in non-[RenderObjectWidget]s
/// and then used in the places mentioned above. Common examples of such widgts
/// are:
///
///   * [InheritedWidget],
///   * [ViewCollection], which allows fitting multiple [View] widgets in a
///     single [view] slot.
///   * [Builder], [StatelessWidget], or [StatefulWidget] that only uses
///     non-[RenderObjectWidget] widgets.
```

Reply:
These docs have been reworked.